package.json

```diff
@@ -29,6 +29,7 @@
     "prettier:all": "prettier --write . --ignore-path .eslintignore",
     "prettier:check": "prettier --check . --ignore-path .eslintignore",
     "proptypes": "cross-env BABEL_ENV=development babel-node -i \"/node_modules/(?!@mui)/\" -x .ts,.tsx,.js ./docs/scripts/generateProptypes.ts",
+    "rsc:build": "tsx ./packages/rsc-builder/buildRsc.ts",
```

Comment:
Shouldn't this be part of CI? 🤔

Reply:
It is not in the core, but it would be a nice addition
I'm trying to add it :+1:

Reply:
It might be a tricky process to do in CI since you'll then want the code to be committed back.
Ideally, it should be part of the build (or did you mean build) ?

Reply:
The CI reminds you to run the script locally anytime you create a new public component (if you did not add "use client" manually).
Similar to what we have for `pnpm l10n`, it does not commit the localization files but it makes sure we don't forget to do it.